src/js/components/products/detail.tsx

```diff
@@ -84,15 +114,35 @@ const ProductDetail = ({ product, productSlug, doFetchObject }: Props) => {
             className="slds-col
               slds-size_1-of-1
               slds-medium-size_2-of-3
-              slds-p-bottom_x-large
-              slds-text-longform"
+              slds-p-bottom_x-large"
           >
-            <Button
-              label={i18n.t('Create a Project')}
-              className="slds-size_full slds-p-vertical_xx-small"
-              variant="brand"
-              disabled
-            />
+            {!projects || !projects.fetched ? (
+              // Fetching projects from API
+              <Spinner />
+            ) : (
+              <>
+                <ProjectForm
+                  product={product}
+                  startOpen={!projects.projects.length}
+                />
+                {Boolean(projects.projects.length) && (
+                  <>
+                    <h2 className="slds-text-heading_medium">
+                      {i18n.t('Projects for')} {product.name}
+                    </h2>
+                    <ul className="slds-has-dividers_bottom">
+                      {projects.projects.map(project => (
+                        <ProjectListItem
+                          key={project.id}
+                          project={project}
+                          product={product}
+                        />
+                      ))}
+                    </ul>
+                  </>
```

Comment:
I think we also want prev/next pagination links here? Could happen in a new branch, since this technically works without it.